From git-commits-head-owner@vger.kernel.org Tue Jan  2 09:02:53 2007
Date: Mon, 1 Jan 2007 19:59:02 GMT
Message-Id: <200701011959.l01Jx2T1002122@hera.kernel.org>
From: Linux Kernel Mailing List <linux-kernel@vger.kernel.org>
To: git-commits-head@vger.kernel.org
Subject: [PATCH] sound: hda: detect ALC883 on MSI K9A Platinum motherboards (MS-7280)

Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=e22a9a8b703d05f13366c3f2e7e1aa0550bb5ca6
Commit:     e22a9a8b703d05f13366c3f2e7e1aa0550bb5ca6
Parent:     36f696cd7ffc82c48bf5826bbb4e005370fb9c6a
Author:     Leonard Norrgård <leonard.norrgard@refactor.fi>
AuthorDate: Mon Jan 1 19:00:54 2007 +0200
Committer:  Linus Torvalds <torvalds@woody.osdl.org>
CommitDate: Mon Jan 1 11:16:32 2007 -0800

    [PATCH] sound: hda: detect ALC883 on MSI K9A Platinum motherboards (MS-7280)
    
    Recognize the Realtek ALC883 chip on MSI K9A Platinum motherboards
    (model no. MS-7280), enabling full sound capabilities.
    
    Signed-off-by: Leonard Norrgård <leonard.norrgard@refactor.fi>
    Signed-off-by: Linus Torvalds <torvalds@osdl.org>
---
 sound/pci/hda/patch_realtek.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index 29e4c48..4e0c3c1 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -5070,6 +5070,8 @@ static struct hda_board_config alc883_cfg_tbl[] = {
 	{ .modelname = "6stack-dig", .config = ALC883_6ST_DIG },
 	{ .pci_subvendor = 0x1462, .pci_subdevice = 0x6668,
 	  .config = ALC883_6ST_DIG }, /* MSI  */
+	{ .pci_subvendor = 0x1462, .pci_subdevice = 0x7280,
+	  .config = ALC883_6ST_DIG }, /* MSI K9A Platinum (MS-7280) */
 	{ .pci_subvendor = 0x105b, .pci_subdevice = 0x6668,
 	  .config = ALC883_6ST_DIG }, /* Foxconn */
 	{ .modelname = "6stack-dig-demo", .config = ALC888_DEMO_BOARD },
